examples/microwave-oven-app/microwave-oven-common/include/microwave-oven-device.h

```diff
@@ -201,16 +225,41 @@ class ExampleMicrowaveOvenDevice : public MicrowaveOvenControl::Delegate,
     ModeBase::Instance mMicrowaveOvenModeInstance;
     MicrowaveOvenControl::Instance mMicrowaveOvenControlInstance;
 
+    // set default value for the optional attributes
+    static constexpr uint8_t kMinPowerNum            = 20u;
+    static constexpr uint8_t kMaxPowerNum            = 110u;
```

Comment:
The is beyond the value allowed by the spec.
```suggestion
    static constexpr uint8_t kMaxPowerNum            = 100u;
```